### PR TITLE
improve translations

### PR DESCRIPTION
**Pull request by retokromer**
While preparing for my teaching in Gorizia, I came across some more nits.

### CODE REVIEW

File: src/Translate.elm [hunk 1]
```diff
@@ -206,8 +206,8 @@ translate language appString =
 
                 ResponsiveStr ->
                     { en = "It looks like you're viewing this on a mobile device or small screen. Sorry, but this app isn't yet mobile-friendly. Please try it on a desktop!"
-                    , fr = "Il paraît que vous utilisez un appareil disposant d'un écran de petite taille. Je suis desolée, mais ce site n'est pas encore mobile conviviale. Veuillez le essayer sur un ordinateur de bureau!"
-                    , it = "Sembra che tu stia utilizzando un dispositivo con uno schermo piccolo. Mi dispiace, questo sito non è ancora mobile-friendly. Per favore lo provare su un desktop!"
+                    , fr = "Il paraît que vous utilisez un appareil disposant d'un écran de petite taille. Je suis desolée, mais ce site n'est pas encore mobile conviviale. Veuillez l'essayer sur un ordinateur de bureau!"
+                    , it = "Sembra che tu stia utilizzando un dispositivo con uno schermo piccolo. Mi dispiace, questo sito non è ancora mobile-friendly. Prova per favove su un desktop!"
```
Comment (kfrn) [inline src/Translate.elm]: Thank you! But is `favove` here a typo for `favore`?

Comment (retokromer) [reply src/Translate.elm]: Of course is! Sorry!

Comment (kfrn) [reply src/Translate.elm]: Haha, all G!

**Comment by kfrn** [comment]
Thanks again!

**Comment by retokromer** [comment]
I was teaching there twice during the last years.

**Pull request merged**